lib/rbi/printer.rb

```diff
@@ -542,6 +542,16 @@ def visit_tenum_block(node)
       printl("end")
     end
 
+    # @override
+    #: (TEnumValue node) -> void
+    def visit_tenum_value(node)
+      print_blank_line_before(node)
+      print_loc(node)
+      visit_all(node.comments)
+
+      printl("#{node.name} = new")
```

Comment:
Again, we might be missing the serialized value of the enum here.

Reply:
Added in `parser_test` instead 👍